### ISSUE TITLE
Kill the pet

### ISSUE THREAD

**Issue by FreshImmuc** [hackdaddy8000/desktop-pet#1]
Hey, please actually release an app that can kill the pet :) (or maybe just implement a function that foes this so everyone can expand it themselves)

**Comment by MrBuushy** [comment]
Yeah! I actually saw the TikTok not too long ago and really wanted to see how he actually made that feature, because that would help me alot in being able to connect an exterior device like a phone to my python code. If he could update it to show that, it would be so much appreciated!
